shared/js/l10n.js

```diff
@@ -1492,6 +1511,7 @@
         return getDirection(navigator.mozL10n.ctx.supportedLocales[0]);
       }
     },
+    gaiaVersion: '2.2',
```

Comment:
Sounds like a good compromise:  we'll use this information only when mozApps is available anyways, so we're already Gecko-specific.